### PR TITLE
Link to front end

### LINKED ISSUE #3: Telegram bot: Provide link to fill an order
**Estimation:** 0.5

- Important, it should consider the fee in order to create the matching order. Validate how we do this so we don't confuse the user
- The url to the frontend should be configurable by env var, and default to localhost

### PR DESCRIPTION
**Pull request by anxolin**
Closes #3 

Creates the link. Please forgive all the TODOs, they are mainly because this project is super affected by the refactor of dex-js, so most of the things are just utils and discussions that should be moved there

This takes into account that the user needs to pay 2*FEE

image

https://dex.dev.gnosisdev.com/trade/DAI-USDC?sell=2.004&buy=1

### CODE REVIEW

File: src/bot.ts [hunk 1]
```diff
@@ -5,8 +5,15 @@ import TelegramBot, { Message, User } from 'node-telegram-bot-api'
 import Logger from 'helpers/Logger'
 import { logUnhandledErrors, onShutdown } from 'helpers'
 import { dfusionService } from 'services'
-import { formatAmount } from 'utils/format'
+import { formatAmount, formatAmountFull } from 'utils/format'
 import BN from 'bn.js'
+import { FEE_DENOMINATOR } from 'const'
+
+const WEB_BASE_URL = process.env.WEB_BASE_URL
+assert(WEB_BASE_URL, 'WEB_BASE_URL is required')
```
Comment (alfetopito) [inline src/bot.ts]: Will probably need the new `assert` @W3stside created on `dex-react`, correct?

File: src/bot.ts [hunk 2]
```diff
@@ -139,18 +146,24 @@ dfusionService.watchOrderPlacement({
...
     // Compose message using markdown
     // TODO: Provide the link to the front end: https://github.com/gnosis/dex-telegram/issues/3
     // TODO: Should we publish even if the user doesn't have balance. Should we include the balance of the user? he can change it...
     const message = `Sell *${sellAmountFmt}* \`${sellTokenLabel}\` for *${buyAmountFmt}* \`${buyTokenLabel}\`
 
   - *Price*:  1 \`${sellTokenLabel}\` = ${price} \`${buyTokenLabel}\`
-${datesDescription}`
+${datesDescription}
+
+Fill the order here: ${WEB_BASE_URL}/trade/${buyTokenLabel}-${sellTokenLabel}?sell=${fillSellAmountFmt}&buy=${buyAmountFullFmt}` // TODO:
```
Comment (alfetopito) [inline src/bot.ts]: Suggestion for a new utils function: `function buildTradeUrl(...)`

Comment (anxolin) [reply src/bot.ts]: issue: https://github.com/gnosis/dex-js/issues/9

File: src/bot.ts (hunk 1 above)
Comment (W3stside) [reply src/bot.ts]: If it's strict mode 3.7.2, yes

Comment (anxolin) [reply src/bot.ts]: Changed the import: 012918a9716ab5bc21a04a0a18db0f1aa70734b0

**Pull request merged**